api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -418,6 +424,31 @@ public void testLocalize_noNotebooks() throws Exception {
     assertThat(resp.getClusterLocalDirectory()).isEqualTo("workspaces/wsid");
   }
 
+  @Test(expected = ForbiddenException.class)
+  public void listCluster_requireActiveBilling() {
+    testWorkspace.setBillingStatus(BillingStatus.INACTIVE);
+
+    doThrow(ForbiddenException.class)
+        .when(workspaceService)
+        .requireActiveBilling(WORKSPACE_NS, WORKSPACE_ID);
+
+    clusterController.listClusters(WORKSPACE_NS, WORKSPACE_ID);
+  }
+
+  @Test(expected = ForbiddenException.class)
+  public void localize_requireActiveBilling() {
+    testWorkspace.setBillingStatus(BillingStatus.INACTIVE);
```

Comment:
Also, I'd probably just make two test workspaces, one active and one not, to cut down on the per-testcase setup.